src/main/webapp/time-range.js

```diff
@@ -237,31 +238,29 @@ const DatePicker = props => {
   const [state, setState] = useState(value)
 
   return (
-    <MuiPickersUtilsProvider utils={DateFnsUtils}>
```

Comment:
Does this break the stories using time-range?

Reply:
Yes, it breaks the time-range stories. Oops.

Reply:
fixed the stories